core/evalir.ml

```diff
@@ -128,6 +128,8 @@ struct
      result =
 
        fun req_data name cont args ->
+         if not(RequestData.is_ajax_call (RequestData.get_cgi_parameters req_data)) then
+           raise (Errors.runtime_error "Tried to make a client call on the server.");
```

Comment:
This should be a call to a new `Error`, similar to `client_call_outside_webmode` that is raised below if a client function is called before the web server is started.

Reply:
Changed to new `Error` type `ClientCallBeforeServerStart`